views/comment.py

```diff
@@ -23,11 +35,20 @@ def get_comments():
         comments = []
 
         for row in query_results:
+            user = {
+                "id": row["id"],
+                "first_name": row["first_name"],
+                "last_name": row["last_name"],
+                "username": row["username"],
+                "email": row["email"],
+            }
             comment = {
                 "id": row["id"],
```

Comment:
By not specifying which `id` is being referenced, this will automatically be set to the most recent `id` (in this case, the `user`'s `id`. This means that when I request the list of comments, each comment has its `id` set to its own `author_id`, which is incorrect. To fix this, you simply need to provide an alias for the `id`s in your query. For example:
```sql
SELECT
    c.id AS comment_id,
    u.id AS user_id
FROM Comments c
JOIN Users u
    on u.id = c.author_id
```

This way, you can later specify `"id": row["comment_id"]` for the comment, and `"id": row["user_id"]` for the user.

Reply:
ok, easy fix